docs/catalog.md

```diff
@@ -370,6 +370,38 @@ Here is an example of creating a package binding and identifying the language of
   }
   ```
 
+### Synthesizes text input to spoken audio
+
+The `/whisk.system/watson/textToSpeech` action help synthesize text input of any supported language to produce audio output. The parameters are as follows:
```

Comment:
action help synthesize text input of any supported language -> action synthesizes text input from a supported language